Server/src/main/java/gotcha/server/Service/Communication/Requests/SetConfigRequest.java

```diff
@@ -0,0 +1,48 @@
+package gotcha.server.Service.Communication.Requests;
+
+/**
+ * LoginRequest is a data transfer object (DTO) that represents the
+ * login request payload containing the email and password for
+ * authentication.
+ */
+
+public class SetConfigRequest {
+    // TODO: 12/04/2023  
+    private String email;
```

Comment:
this is the email of the person who is calling the set_config? If so then this will already be part of the UserContext and can be accesed from there